llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -1609,3 +1609,28 @@ bool RISCVTTIImpl::isLSRCostLess(const TargetTransformInfo::LSRCost &C1,
                   C2.NumIVMuls, C2.NumBaseAdds,
                   C2.ScaleCost, C2.ImmCost, C2.SetupCost);
 }
+
+bool RISCVTTIImpl::isLegalMaskedCompressStore(Type *DataTy, Align Alignment) {
```

Comment:
I think this belongs in the patch that adds the RISCVISelLowering support. It's not legal for RISC-V without that.